### ISSUE TITLE
Read.py Prompt Engineering Error

### ISSUE THREAD

**Issue by nulflux** [neonbjb/tortoise-tts#65]
Attempting to use prompt engineering with Read.py leads to this error:

Traceback (most recent call last):
  File "tortoise/beyond.py", line 68, in <module>
    gen = tts.tts_with_preset(text, voice_samples=voice_samples, conditioning_latents=conditioning_latents,
  File "/media/5TB/Env/TortoiseTTS/tortoise-tts/tortoise/api.py", line 302, in tts_with_preset
    return self.tts(text, **kwargs)
  File "/media/5TB/Env/TortoiseTTS/tortoise-tts/tortoise/api.py", line 451, in tts
    wav_candidates = [potentially_redact(wav_candidate, text) for wav_candidate in wav_candidates]
  File "/media/5TB/Env/TortoiseTTS/tortoise-tts/tortoise/api.py", line 451, in <listcomp>
    wav_candidates = [potentially_redact(wav_candidate, text) for wav_candidate in wav_candidates]
  File "/media/5TB/Env/TortoiseTTS/tortoise-tts/tortoise/api.py", line 449, in potentially_redact
    return self.aligner.redact(clip.squeeze(1), text).unsqueeze(1)
  File "/home/beyond/.local/share/virtualenvs/TortoiseTTS-3YyTW6Mb/lib/python3.8/site-packages/TorToiSe-2.4.0-py3.8.egg/tortoise/utils/wav2vec_alignment.py", line 141, in redact
  File "/home/beyond/.local/share/virtualenvs/TortoiseTTS-3YyTW6Mb/lib/python3.8/site-packages/TorToiSe-2.4.0-py3.8.egg/tortoise/utils/wav2vec_alignment.py", line 105, in align
AssertionError: Something went wrong with the alignment algorithm. I've dumped a file, 'alignment_debug.pth' toyour current working directory. Please report this along with the file so it can get fixed.

Is there any fix for this?
[alignment_debug.pth.zip](https://github.com/neonbjb/tortoise-tts/files/8773424/alignment_debug.pth.zip)

**Comment by nulflux** [comment]
Apparently I have two different folders with TTTS - here's the most current debug file.
[alignment_debug.pth.zip](https://github.com/neonbjb/tortoise-tts/files/8773460/alignment_debug.pth.zip)
)

**Comment by neonbjb** [comment]
I'll need to do some debugging, I may get some time tomorrow night to take a look.

**Comment by jd-3d** [comment]
I also get this error when prompt engineering. I think it broke with the 5/12 update (which is when I first noticed it).

**Comment by nulflux** [comment]
Last night I did some testing: prompt engineering does work in some scenarios and not in others. For instance a text file with a single line and an engineered prompt works every time I tested it. I can add up to 4 on the same line. If I go beyond 4 it has an alignment error. I also tested 3 on one line, 3 on the second line - it errors after the 4th. I tested with | dividing the phrases and same issue. However - if I just grab a solid paragraph from a normal book and shove it into the text file it works like a champ. It seems to handle other types of braces just fine (){}** etc. yet that doesn't achieve the desired result - it will 'say' the modifier instead of using it.
